### DISCUSSION TITLE
Import custom fields form csv import

### DISCUSSION THREAD

**Discussion by bubfcg** [code4recovery/12-step-meeting-list#1413] [Q&A] [answered]
I have created 2 custom fields that I want to use to write SEO headlines. I have added the custom fields to 1 meeting in the WordPress backend and added code to my theme's function.php to add those fields to the export file.

I was hoping that I could fill the custom fields for all the other meetings in Excel and import the csv file, but that didn't work. Is there any way to this, so that the custom fields are generated and filled for all the meetings by simply importing the csv file?

And how can I update meetings that already exist on my website and have an ID in my csv file? Because what happened now after uploading is that they were added as new meetings. Should I use the import option 'delete only the meetings, locations, and groups for the regions present in this CSV'?

**Comment by joshreisner** [comment]
going to move this to the right place. tsml ui is the frontend, this belongs in the plugin discussions

**Reply by bubfcg** [reply]
Ah, my bad. Apologies.

**Reply by joshreisner** [reply]
no worries! it's pretty confusing

**Comment by joshreisner** [comment]
so it sounds like there are two issues here:

* want to import custom fields
this is something we haven't programmed yet but seems like it should be feasible. we're in the middle of a big revamp to one part of the importer so this isn't likely to get implemented right away

* want to import and update existing meetings
i need to understand a little more about your workflow - where does the CSV you're importing come from? was it imported from your site or another site or a google sheet? does it have all the meetings on your site or only a subset?

**Comment by bubfcg** [comment]
Hey @joshreisner 

- I have created the custom fields with ACF, want to fill the data with an import
- Created the export form the website, with all the meetings in it

**Comment by kiyote33** [comment]
Hi @bubfcg,
We've created a feature request issue based on this discussion (#1416) to include the exported ACF fields in the Importer process. Tentative plan is to include it in an upcoming maintenance release of our 12 Step Meeting List plugin.

**Reply by bubfcg** [reply]
Awesome @kiyote33! Thanks for the great work and support! Will it then also be possible to update the ACF fields for meetings that are already on the website using the import function?

**Reply by kiyote33** [reply]
Yes, that's the plan.

**Reply by kiyote33** [reply, marked as answer]
This feature is now available in the 3.14.35 Release.
